justfile

```diff
@@ -44,6 +44,8 @@ _tools-linter:
 
 # Other packages should use the _install-go-pkg script.
 _tools-generate:
+    @just _install-go-pkg "google.golang.org/grpc/cmd/protoc-gen-go-grpc"
+    @just _install-go-pkg "google.golang.org/protobuf" "cmd/protoc-gen-go"
```

Comment:
I don't think anything should require grpc here? but definitely possible protocol is required.

Reply:
`just generate` doesnt currently run in the repo. this is kinda unrelated to this change but i cant regenerate the test cases without fixing this